src/helix-ui/elements/index.js

```diff
@@ -21,8 +21,9 @@ export { HXModalElement } from './HXModalElement';
 export { HXPillElement } from './HXPillElement';
 export { HXPopoverElement } from './HXPopoverElement';
 export { HXProgressElement } from './HXProgressElement';
-export { HXRadioControlElement } from './HXRadioControlElement';
 export { HXRadioElement } from './HXRadioElement';
+export { HXRadioControlElement } from './HXRadioControlElement';
```

Comment:
Please alphabetize. `HXRadioControlElement` would appear before `HXRadioElement` in alphabetical order.

Reply:
reverted order